client/components/thank-you/index.tsx

```diff
@@ -151,26 +147,20 @@ export const ThankYou = ( props: ThankYouProps ): JSX.Element => {
 			<ThankYouHeader className={ classNames( 'thank-you__container-header', headerClassName ) }>
 				{ /* eslint-disable-next-line jsx-a11y/alt-text */ }
 				<img { ...thankYouImage } />
-				<ThankYouFooterContainer>
-					<h1> { thankYouImageFooter } </h1>
-					<h2> { thankYouImageFooterSubtitle } </h2>
-				</ThankYouFooterContainer>
+				{ thankYouTitle && (
+					<ThankYouTitleContainer>
+						<h1 className="thank-you wp-brand-font"> { thankYouTitle } </h1>
+						<h2> { thankYouSubtitle } </h2>
```

Comment:
Two comments:
1. I think we should make this conditional on `thankYouSubtitle` being defined -- otherwise you end up with `<h2></h2>` in the HTML which can cause spacing trouble due to margin/padding etc for the `h2` element.
2. I'd prefer if we remove the whitespace outside the `{}` wrapper. (This gives `<h2>  </h2>` when it's empty at present.)
```suggestion
						{ thankyouSubtitle && <h2>{ thankYouSubtitle }</h2> }
```